src/test/java/it/com/atlassian/bitbucket/jenkins/internal/config/BitbucketProjectConfigurationIT.java

```diff
@@ -46,10 +55,28 @@ public void tearDown() throws IOException, InterruptedException {
         project.delete();
     }
 
+    @Test
+    public void testBitbucketSCMFieldsShouldBePopulatedWithProperValues() throws IOException, SAXException {
+        setupBitbucketSCM();
+
+        HtmlPage configurePage = webClientRule.visit("job/" + JENKINS_PROJECT_NAME + "/configure");
+        HtmlForm form = configurePage.getFormByName("config");
+
+        HtmlSelect credential = form.getSelectByName("_.credentialsId");
+        waitTillItemIsRendered(credential::getOptions);
+        assertEquals(bbJenkinsRule.getBitbucketServerConfiguration().getCredentialsId(), credential.getSelectedOptions().get(0).getValueAttribute());
+
+        HtmlSelect serverId = form.getSelectByName("_.serverId");
+        waitTillItemIsRendered(serverId::getOptions);
+        assertEquals(bbJenkinsRule.getBitbucketServerConfiguration().getId(), serverId.getSelectedOptions().get(0).getValueAttribute());
+
+        assertEquals(PROJECT_NAME, form.getInputByName("_.projectName").getValueAttribute());
```

Comment:
Not your code, but I think we should be using `BitbucketUtils.PROJECT_NAME` here rather than having a separate set of static fields (and in the other areas that applies)